portfolio/src/main/webapp/addclass.css

```diff
@@ -0,0 +1,91 @@
+body {
+    background-color: white;
+  }
+  
+  .form-para {
+    color: black;
+    font-size: 3rem;
+    text-align: left;
+    padding-top: 50px;
+  }
+  
+  .input-header {
+    color: black;
+    font-size: 1rem;
+    text-align: left;
+    padding-bottom: 10px;
+    padding-top: 10px;
+  }
+  
+  .btn {
+    margin-top: 20px;
+    background-color: black;
+  }
+  
+  .design-quote {
+    color: black;
+    font-size: 1.5rem;
+    text-align: left;
+    padding-top: 50px;
+  }
+  
+  ::-webkit-input-placeholder {
+    /* Chrome/Opera/Safari */
+    color: pink;
+    font-family: "Playfair Display", serif;
+  }
+  ::-moz-placeholder {
+    /* Firefox 19+ */
+    color: pink;
+    font-family: "Playfair Display", serif;
+  }
+  :-ms-input-placeholder {
+    /* IE 10+ */
+    color: pink;
+    font-family: "Playfair Display", serif;
+  }
+  :-moz-placeholder {
+    /* Firefox 18- */
+    color: pink;
+    font-family: "Playfair Display", serif;
+  }
+  
+  .design-phil {
+    font-family: "Playfair Display", serif;
+    color: black;
+    font-size: 1rem;
+    text-align: left;
+    padding-top: 50px;
+  }
+  
+  .brand {
+    color: black;
+    font-size: 2rem;
+    text-align: center;
+  }
+  
+  .drop {
+    color: black;
+    font-size: 1rem;
+    text-align: center;
+  }
+  
+  .intro {
+    color: black;
+    font-size: 3rem;
+    text-align: left;
+    padding-top: 100px;
+  }
+  
+  .intro-sen {
+    color: white;
+    font-size: 1.5rem;
+    text-align: left;
+    padding-top: 30px;
+    padding-bottom: 40rem;
+  }
+  
+  .mobile-button {
+    color: white;
+  }
+  
```

Comment:
how about adding some paddings in addClass.html at the bottom? The contact form is too closed from bottom. Also, maybe we could let the form to be in the middle? It will look nicer speaking from the perspective of a reader.